FastSurferCNN/utils/metrics.py

```diff
@@ -103,175 +107,317 @@ def precision_recall(
 
 class DiceScore:
     """
-    Accumulate the component of the dice coefficient i.e. the union and intersection.
-
-    Attributes
-    ----------
-    op : callable
-        A callable to update accumulator. Method's signature is `(accumulator, output)`.
-        For example, to compute arithmetic mean value, `op = lambda a, x: a + x`.
-    output_transform : callable, optional
-        A callable that is used to transform the
-        :class:`~ignite.engine.Engine`'s `process_function`'s output into the
-        form expected by the metric. This can be useful if, for example, you have a multi-output model and
-        you want to compute the metric with respect to one of the outputs.
-    device : str of torch.device, optional
-        Device specification in case of distributed computation usage.
-        In most of the cases, it can be defined as "cuda:local_rank" or "cuda"
-        if already set `torch.cuda.set_device(local_rank)`. By default, if a distributed process group is
-        initialized and available, device is set to `cuda`.
+    Accumulating the component of the dice coefficient i.e. the union and intersection.
+
+    Args:
+        op (callable): a callable to update accumulator. Method's signature is `(accumulator, output)`.
+            For example, to compute arithmetic mean value, `op = lambda a, x: a + x`.
+        output_transform (callable, optional): a callable that is used to transform the
+            :class:`~ignite.engine.Engine`'s `process_function`'s output into the
+            form expected by the metric. This can be useful if, for example, you have a multi-output model and
+            you want to compute the metric with respect to one of the outputs.
+        device (str of torch.device, optional): device specification in case of distributed computation usage.
+            In most of the cases, it can be defined as "cuda:local_rank" or "cuda"
+            if already set `torch.cuda.set_device(local_rank)`. By default, if a distributed process group is
+            initialized and available, device is set to `cuda`.
```

Comment:
Fix this docstring so it is also Numpy-style
```
Parameters
----------
ope : callable
    A callable ...
...
```

Reply:
Done